src/idpi/operators/relhum.py

```diff
@@ -0,0 +1,45 @@
+"""Relative humidity operators."""
+# Standard library
+from typing import Literal
+
+# Third-party
+import xarray as xr
+
+# First-party
+from idpi.operators.atmo import pv_sw
+from idpi.operators.atmo import qv_pvp
+
+
+def relhum(
+    qv, t, p, clipping=True, phase: Literal["water", "ice", "water+ic"] = "water"
+):
+    """Calculate relative humidity.
+
+    Parameters
+    ----------
+    qv : xr.DataArray
+        waver vapor mixing ratio
+    t : xr.DataArray
+        temperature
+    p : xr.DataArray
+        pressure
+    clipping : bool
+        clips the relative humidity to [0,100] interval
```

Comment:
should note that only the upper bound is controlled by this parameter

Reply:
done